lib/arjdbc/postgresql/base/pgconn.rb

```diff
@@ -1,8 +1,8 @@
-module ActiveRecord::ConnectionAdapters::PostgreSQL::OID
-  class PGconn # emulate PGconn#unescape_bytea due #652
```

Comment:
@enebo is PG all over the place in 5.0 at AR's, wasnt that hard to support over the years :)?
... you could still use the trick of not polluting by putting PG under a common name-space
e.g. `ActiveRecord::ConnectionAdapters::PostgreSQLAdapter::PG` have used similar around here

I do not have anything off hand, its been a while - in general AR-JDBC has code in at least 3 top level name-spaces, so this 4-th seemed like it can be avoided ... but I have no strong objections if its to be kept. unless I get "hired" for AR-JDBC tech.debt elimination :)